pkg/settings/setting.go

```diff
@@ -91,6 +91,7 @@ var (
 	UIDashboardIndex                    = NewSetting("ui-dashboard-index", "https://releases.rancher.com/dashboard/latest/index.html")
 	UIDashboardPath                     = NewSetting("ui-dashboard-path", "/usr/share/rancher/ui-dashboard")
 	UIFavicon                           = NewSetting("ui-favicon", "")
+	UIPerformance                       = NewSetting("ui-performance", "") // Experimental settings for UI functionality to improve the UX with large nunbers of resources
 	UIPreferred                         = NewSetting("ui-preferred", "vue")
 	UIOfflinePreferred                  = NewSetting("ui-offline-preferred", "dynamic")
 	UIIssues                            = NewSetting("ui-issues", "")
```

Comment:
I'd like to move all these UI settings to the ordered settings list below and have more formal comments around what they do and how, but that can happen in a follow up PR.